LTS-CHANGELOG.adoc

```diff
@@ -20,6 +20,8 @@ All fixes and changes in LTS releases will be released the next minor release. C
 [[v1.6.30]]
 == 1.6.30 (TBD)
 
+icon:plus[] Auth: Added JWT options to configure token content (`issuer` - `"iss"`, `audience` - `"aud"`) and token verification (`leeway` and `ignoreExpiration`).
```

Comment:
You're a bit out of date with the origin branch, latest relased version is 1.6.34, so this should be 1.6.35